h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -241,6 +242,18 @@ private Frame prepareLevelOneFrame(String levelOneKey, Key<Model>[] baseModelKey
       return levelOneFrame;
     }
 
+    private boolean shouldStop() {
+      if (stop_requested()) {
+        if (timeout()) {
+          Log.info("Stopping StackedEnsemble training because of timeout");
+          return true;
```

Comment:
why just breaking the loop in case of timeout, will it not keep the training with only a subset of base models?